sdk/servicebus/Azure.Messaging.ServiceBus/README.md

```diff
@@ -338,6 +338,8 @@ string reason = dlqMessage.DeadLetterReason;
 string description = dlqMessage.DeadLetterErrorDescription;
 ```
 
+For more information, see the [overview of ServiceBus dead letter queues](/azure/service-bus-messaging/service-bus-dead-letter-queues).
```

Comment:
```suggestion
For more information, see the [overview of ServiceBus dead letter queues](https://learn.microsoft.com/azure/service-bus-messaging/service-bus-dead-letter-queues).
```

We'll need to make this an absolute URL, as the content will be used from both the GitHub repository and MS Learn.

Reply:
Gotcha 👍🏼